sgqlc/types/__init__.py

```diff
@@ -1954,8 +2020,12 @@ def type(self):
     def _to_graphql_name(name):
         '''Converts a Python name, ``a_name`` to GraphQL: ``aName``.
```

Comment:
I would also mention that this preserves the `_` prefixes.

Reply:
nice catch